jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -716,3 +717,26 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_ha(){
```

Comment:
please add bats tests for this function.

Reply:
ok